softlayer/resource_softlayer_global_ip.go

```diff
@@ -45,6 +47,16 @@ func resourceSoftLayerGlobalIp() *schema.Resource {
 			"routes_to": &schema.Schema{
 				Type:     schema.TypeString,
 				Required: true,
+				ValidateFunc: func(v interface{}, k string) (ws []string, errors []error) {
+					address := v.(string)
+					if net.ParseIP(address) == nil {
+						errors = append(errors, fmt.Errorf("Invalid IP"))
```

Comment:
Be more specific with the error `fmt.Errorf("Invalid IPv6 format: %s", address)`